waku/factory/node_factory.nim

```diff
@@ -422,7 +422,11 @@ proc startNode*(
       desiredOutDegree = node.wakuRelay.parameters.d.uint64()
     (await node.fetchPeerExchangePeers(desiredOutDegree)).isOkOr:
       error "error while fetching peers from peer exchange", error = error
-      quit(QuitFailure)
+
+  # Use px to periodically get peers if discv5 is disabled, as discv5 nodes have their own
+  # periodic loop to find peers and px returned peers actually come from discv5
+  if conf.peerExchange and not conf.discv5Discovery:
```

Comment:
I tested it even without setting `peerExchangeNode ` and worked :))

If there isn't a `peerExchangeNode ` set, then any peer we have in the store with the PX codec is chosen randomly, so we just need to be connected to at least one peer that supports PX
https://github.com/waku-org/nwaku/blob/b3656d6eeaacdc0af5b479a17d837616930411e4/waku/node/peer_manager/peer_manager.nim#L243-L254

It can be any of our normal bootstrap nodes or a static node also.

Then, the peers we receive via PX if they also support it will enter to the pool from which we choose randomly, so it's just a matter of having a point where to start from, and luckily it must not necessarily be a configured `peerExchangeNode` :)